src/Symfony/Bundle/FrameworkBundle/Client.php

```diff
@@ -104,6 +106,45 @@ public function enableReboot()
         $this->reboot = true;
     }
 
+    /**
+     * Logins a user against a specific firewall.
+     *
+     * @param string|object $user
+     * @param string        $credentials
```

Comment:
this is a `mixed` argument in `UsernamePasswordToken`.

And anyway, credentials must be erased before serialization to match what is done in the security component when saving the token in the session. so you don't even need to ask for the credentials here (they will be `null` in the saved token)

Reply:
I removed the parameter